doc/src/modules/assumptions/index.rst

```diff
@@ -32,7 +32,7 @@ assumptions about objects in expr. Valid values include:
     * Q.integer(x) & Q.positive(x)
     * etc.
 
-Q is a class in sympy.assumptions holding known predicates.
+Q is an instance of QClass in sympy.assumptions.ask holding known predicates.
```

Comment:
I don't know what `QClass` does, so I can't recommend a name. It should reflect what an object of that class is good for, so `FooClass` is almost always a bad name, that's all I know.
`Keys` is often too generic. Unless objects of that type are really just arbitrary keys and nothing else, that is.
BTW I suspect that users aren't supposed to instantiate objects of class `Q`; if that is correct, it should be documented as `Q is a singleton object`. Also, if users aren't supposed to import it from `sympy.assumptions` but from `sympy`, the reference to the package should be removed; this is user-facing documentation, and what's where is an implementation detail (and we do reorganize packages after all).